cogs/anime/__init__.py

```diff
@@ -566,7 +566,19 @@ async def activity(self, ctx: Context, user: AniUserConv = aniuser):
             embed = PrimaryEmbed()
             embed.set_author(name=f"{name}'s recent activity", icon_url=activities[0]["user"]["avatar"]["large"])
 
-            fmtd = [f"-# {discord.utils.format_dt(act[1], 'R')}\n{act[0]}" for act in chunk]
+            fmtd = [
+                f"-# {discord.utils.format_dt(act[1], 'R')} "
+                + " **|** ".join(
+                    _
+                    for _ in [
+                        (f"\N{WHITE HEART SUIT} {act[2]}" if act[2] > 0 else None),
+                        (f"\N{LOWER RIGHT PENCIL} {act[3]}" if act[3] > 0 else None),
+                    ]
```

Comment:
```suggestion
                    for _ in (
                        (f"\N{WHITE HEART SUIT} {act[2]}" if act[2] > 0 else None),
                        (f"\N{LOWER RIGHT PENCIL} {act[3]}" if act[3] > 0 else None),
                    )
```
Can you make this a tuple? (Promise this is the last one)